cypress/e2e/specialFeatures/cookieBanner/index.cy.js

```diff
@@ -44,7 +44,9 @@ const urlsToExcludeFromAmpTests = ['_tv', '_radio', '/20'];
 Object.keys(config)
   .filter(service => serviceFilter(service))
   .forEach(service => {
-    const { variant } = config[service];
+    const { variant, serviceName } = config[service];
```

Comment:
Instead of `serviceName` we can just use `service` here and a couple of lines down in the `isHomePagePath` function.